breathecode/marketing/tests/mixins/marketing_test_case.py

```diff
@@ -86,21 +87,28 @@ def check_email_contain_a_correct_token(self, lang, dicts, mock, model):
         self.assertToken(token)
         self.assertTrue(link in html)
 
-    # This function is incompleted
     def check_old_breathecode_calls(self, mock, model):
-        self.assertEqual(mock.call_args_list, [
-            call(
-                'POST',
-                'https://old.hardcoded.breathecode.url/api/3/contactAutomations',
-                headers={
+        mock_list = list(mock.call_args_list[1])
+        del mock_list[1]['json']['contactAutomation']['automation']
```

Comment:
The external apis like google can't be use in a test, we instead of call the original api, we'll make a minimal copy of api, when this copy is called it save the params inside `call_args_list`, the only way to we be sure it works in production is check it is being called correctly